packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -1092,6 +1096,77 @@ class VmServiceWrapper implements VmService {
         parser: ObjectStore.parse,
       );
 
+  Future<dap.VariablesResponseBody?> dapVariablesRequest(
+    dap.VariablesArguments args,
+  ) async {
+    final response = await _sendDapRequest(
+      dap.Request(
+        command: 'variables',
+        seq: 0,
```

Comment:
We don't currently rely on it in the adapter, but `seq` is specified as an increasing/unique number so I'd make a field for this and use `_seq++` in these calls.

(maybe `_sendDapRequest` could just take the `command` and `args` and automatically unwrap the response to avoid some boilerplate in each of these?)

Reply:
Ah good to know! Switched to incrementing it, and moving some of the boilerplate (plus adding error handling) to `_sendDapRequest` method